1.20.5/1.20.5-pre2.md

```diff
@@ -0,0 +1,15 @@
+advancement | `Snip it!` advancement is renamed to `Shear Brilliance`
+
+data | Using ctrl+pick block no longer lists "(+NBT)" in the item tooltip
```

Comment:
This should have categories `item` and `nbt`. `data` is for changes to the builtin vanilla data pack.

Reply:
fixed. thank you for all the feedback!